package.json

```diff
@@ -43,14 +43,16 @@
     "lint-js": "eslint .",
     "lint-php": "composer lint",
     "lint": "npm run lint-js && npm run lint-php",
-    "cli": "./local/vagrant/cli.sh",
-    "compose": "./local/vagrant/docker-compose.sh",
+    "stop-all": "docker stop $(docker ps -a -q)",
```

Comment:
This is a new helper to stop all running containers on a Docker host in case there are port conflicts.